### PR TITLE
api: add docs for targets and alertmanagers api

### LINKED ISSUE #649: Document target and alertmanager API endpoints
@brancz based on your last PRs this needs a brief mention in the HTTP API doc section.

### PR DESCRIPTION
**Pull request by brancz**
Short docs on the recently added targets and alertmanagers API.

@fabxc @brian-brazil 

Closes #649

### CODE REVIEW

File: content/docs/querying/api.md [hunk 1]
```diff
@@ -337,3 +337,73 @@ String results are returned as result type `string`. The corresponding
 ```
 [ <unix_time>, "<string_value>" ]
 ```
+
+
+## Targets
+
+> This API is in beta as it is intended to be extended with targets dropped due
```
Comment (brian-brazil) [inline content/docs/querying/api.md]: experimental is the wording we've used thus far

Comment (fabxc) [inline content/docs/querying/api.md]: Actually we used `experimental` and `beta` in the past depending on the stability degree. Experimental sounds good though for the very first release.

**Comment by fabxc** [comment]
LGTM in general 👍

File: content/docs/querying/api.md (hunk 1 above)
Comment (brancz) [inline content/docs/querying/api.md]: fixed

**Pull request merged**